cli/hardhat.config.ts

```diff
@@ -17,7 +17,7 @@ const config: HardhatUserConfig = {
     localhost: {
       url: process.env.ETH_PROVIDER || DEFAULT_ETH_PROVIDER,
       accounts: [process.env.ETH_SK || DEFAULT_ETH_SK],
-      loggingEnabled: false,
+      loggingEnabled: true,
```

Comment:
let's maybe keep logging to false, and if someone needs they can change to true?